homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -101,14 +102,23 @@ class SwissPublicTransportSensor(
 
     _attr_attribution = "Data provided by transport.opendata.ch"
     _attr_icon = "mdi:bus"
+    _attr_has_entity_name = True
 
     def __init__(
-        self, coordinator: SwissPublicTransportDataUpdateCoordinator, name: str
+        self,
+        coordinator: SwissPublicTransportDataUpdateCoordinator,
+        unique_config_title: str,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._coordinator = coordinator
-        self._attr_name = name
+        self._attr_name = "Departure"
```

Comment:
While you're at it you can also add translations. Add `_attr_translation_key = "departure"` (can be outside of the constructor) and add
```json
"entity": {
  "sensor": {
    "departure": {
      "name": "Departure"
    }
  }
}
```
to strings.json